src/iam/chainConfig.ts

```diff
@@ -10,7 +10,7 @@ import { MessagingMethod } from "../utils/constants";
 
 const VOLTA_CHAIN_ID = 73799;
 export interface ChainConfig {
-  rpcUrl: string;
+  rpcUrl: string,
```

Comment:
What is the reason of changing separator?

Reply:
just a typo :/  fixed with a semicolon, strange that even with a comma it passed  tests and built successfully, maybe interface syntax Is supporting both commas and semicolons